chainquery/compose/docker-compose.yml-prod-example

```diff
@@ -8,16 +8,13 @@ services:
 ###########
 ## MYSQL ##
 ###########
-## MariaDB is currently not supported and neither is later versions of MySQL this may change.
-## https://hub.docker.com/r/_/mariadb/
+## Work is underway to support percona
```

Comment:
Has this been decided proper at this point I spoke with @nikooo777 a month or so ago and was told not to bother.

Reply:
Also thank you for the cleanup.